README.md

```diff
@@ -85,9 +91,17 @@ WD-Firmware
 	- In the Import window select *General* -> *Existing Projects into Workspace* -> *Next*
 	- In the next Import window *Select root directory* -> *Browse*  and choose the infra-riscv-fw/WD-Firmware/ you've downloaded in 'Getting the firmware sources' section
 	- Press *'Finish'* button
-- From 'Eclipse MCU' menu bar select '*Project'* -> *'Build All'*
+- From 'Eclipse MCU' menu bar select '*Project'* -> *'Build All'*. Note that you can select which platform to build for.
+- You you will need to choose a specific ***demo*** for building a full sultion:
+
+        from the eclipse terminal or console:
+        $ cd [WD-firmware-root]/WD-Firmware/demo/build
+        $ python ./config.sh

```

Comment:
You need to run ./config.sh, not as written here - python ./config.sh

Reply:
done